src/isomorphic/classic/element/ReactElementValidator.js

```diff
@@ -194,8 +194,9 @@ var ReactElementValidator = {
         false,
         'React.createElement: type should not be null, undefined, boolean, or ' +
           'number. It should be a string (for DOM elements) or a ReactClass ' +
-          '(for composite components).%s',
-        getDeclarationErrorAddendum()
+          '(for composite components).%s%s',
+        getDeclarationErrorAddendum(),
+        ReactComponentTreeHook.getCurrentStackAddendum()
```

Comment:
I've tried passing on `ReactCurrentOwner.current` but that doesn't work since it's a `ReactInstance` but it expects a `ReactElement`. Is there any way I can get `ReactElement` from the `ReactInstance`? :thinking: